src/ethereum-json-rpc-client/Cargo.toml

```diff
@@ -15,13 +15,15 @@ pocket-ic-tests-client = [
   "ic-canister-client/pocket-ic-client",
 ]
 reqwest = ["dep:reqwest"]
+http-outcall = ["dep:url"]
 
 [dependencies]
 anyhow = { workspace = true }
 candid = { workspace = true }
 hex = { workspace = true }
 ic-exports = { workspace = true }
 ic-canister-client = { workspace = true, optional = true }
+url = { workspace = true, optional = true }
```

Comment:
why do you need to import `url` explicitly? `reqwest::Url` is a re-export of it

Reply:
The requwest have problems with building into wasm(